prusa/connect/printer/cameras.py

```diff
@@ -412,6 +500,11 @@ def __init__(self, event_cb: EventCallback):
             TriggerScheme.TEN_MIN: self._was_10_min,
             TriggerScheme.EACH_LAYER: self._was_layer_change
         }
+        self.snapshot_loop_thread = Thread(target=self.snapshot_loop,
```

Comment:
Are there any other Threads in SDK? I think that better place for thread is Link, but if there are some Threads yet, that is ok.

Reply:
There aren't any threads yet, apart from the ones for taking pictures. Moving to MyPrinter